### PR TITLE
SCRUM-8 post_create_task

### PR DESCRIPTION
**Pull request by FreddyFZ17**
The new smokes testcases were added to the "post_create_task" endpoint where 4 smokes testcases were created and 4 that were not, but they were not fully implemented. The new zetup and teardown functionalities were also implemented for the testcases that require it.

### CODE REVIEW

**Review by Jasonn5 — APPROVED** [review]
Looks good, executed successfully, good job Fredy

File: api_endpoints/api_request.py
Comment (AlisonGuzman2006) [inline api_endpoints/api_request.py:1]: This addition int he code has ben reviewed and approved

File: api_endpoints/task_ednpoints.py
Comment (AlisonGuzman2006) [inline api_endpoints/task_ednpoints.py:1]: This addition int he code has ben reviewed and approved

File: assertions/assertion_schemas.py
Comment (AlisonGuzman2006) [inline assertions/assertion_schemas.py:1]: This addition in the code has ben reviewed and approved

File: payloads/tasks/payload_schema_task.json
Comment (AlisonGuzman2006) [inline payloads/tasks/payload_schema_task.json:1]: This addition in the code has ben reviewed and approved

File: payloads/tasks/payload_tasks.py
Comment (AlisonGuzman2006) [inline payloads/tasks/payload_tasks.py:1]: This addition in the code has ben reviewed and approved

File: tests/task/conftest.py
Comment (AlisonGuzman2006) [inline tests/task/conftest.py:1]: This addition in the code has ben reviewed and approved

File: tests/task/test_post_create_task.py
Comment (AlisonGuzman2006) [inline tests/task/test_post_create_task.py:1]: This addition in the code has ben reviewed and approved

**Review by AlisonGuzman2006 — APPROVED** [review]
These changes have been approved

**Pull request merged**